src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,35 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int NUMBER_OF_FIGURES = 5;
+    private Random random = new Random();
+    private ColorSupplier colorSupplier = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        int figureIndex = random.nextInt(NUMBER_OF_FIGURES);
+        String color = colorSupplier.getRandomColor().toLowerCase();
```

Comment:
The `colorSupplier.getRandomColor().toLowerCase()` line converts the color to lowercase. Ensure that this is intentional and that the rest of the application handles lowercase color names correctly.